db/migrate/20230415183210_create_answer_fun_questions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAnswerFunQuestions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :answer_fun_questions do |t|
```

Comment:
Model names should be nouns rather than verbs and since this is an open source project (with other developers looking at this code in the future) it would be good to stay within these kinds of conventions. Maybe `fun_question_response` or `fun_question_answer` would be better options?